UPGRADE-7.2.md

```diff
@@ -73,6 +73,7 @@ String
    * `TruncateMode::Char` is equivalent to `true` value ;
    * `TruncateMode::WordAfter` is equivalent to `false` value ;
    * `TruncateMode::WordBefore` is a new mode that will cut the sentence on the last word before the limit is reached.
+ * Add `kebab()` method to `AbstractString`
```

Comment:
```suggestion
   * Add `kebab()` method to `AbstractString`
```

Reply:
The items just above this new line are indented with another item. The newline only has one space indentation like it should

Reply:
Oh, got baited by GitHub UI, my bad